slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -141,6 +141,15 @@ class SlideToActView @JvmOverloads constructor(
             invalidate()
         }
 
+    /** Custom Icon */
+    var icon: Int = R.drawable.slidetoact_ic_arrow
```

Comment:
```suggestion
    var sliderIcon: Int = R.drawable.slidetoact_ic_arrow
```

That's basically to:

- Be consistent with the `slider_icon` XML property
- Allow for future expansion for a potential `tick_icon`/`tickIcon` (see #14)

Reply:
Have included this change in my latest commit.